megamek/src/megamek/client/ui/swing/util/KeyCommandBind.java

```diff
@@ -77,6 +77,7 @@ public enum KeyCommandBind {
     /** Used to cancel moves/fires/chatterbox */
     CANCEL("cancel", false, VK_ESCAPE, 0, true),
     DONE("done", false, VK_ENTER, CTRL_DOWN_MASK, true),
+    DONE_NO_ACTION("doneNoAction", false, VK_ENTER, CTRL_DOWN_MASK | SHIFT_DOWN_MASK, true),
```

Comment:
Please add an entry for this to mmconf/defaultkeybinds.xml. We've had fresh MM downloads fail to run because the xml entry for a new keybind was missing even though it worked perfectly in the IDE.

Reply:
@SJuliez How do I add a binding that has multiple modifier keys, in this case "ctrl+shift enter?"